entity_event_generator.tf

```diff
@@ -0,0 +1,56 @@
+locals {
+  entity_event_lambda_name = "${local.environment}-entity-event-lambda"
```

Comment:
`entity-event-lambda` or `entity-event-generator-lambda`? I know the file is the latter which matches the repo for the lambda code, should the modules and resources also match this to make it easier to track what the lambda is within a larger application?